src/WooCommerce.php

```diff
@@ -1494,6 +1494,13 @@ public static function woocommerce_save_account_details($user_id) {
       wc_add_notice(isset($response['userMessages']) ? implode('<br>', $response['userMessages']) : __('Error while saving the changes.'), 'error');
       return;
     }
+    // Display message to log in again after changing display name.
+    if ((!empty($_POST['account_first_name']) && $_POST['account_first_name'] !== static::$userBeforeUpdate->first_name)
+     || (!empty($_POST['account_last_name']) && $_POST['account_last_name'] !== static::$userBeforeUpdate->last_name)) {
+       wc_add_notice(vsprintf('Damit Ihre Änderungen wirksam werden, <a href="%s">melden Sie sich bitte ab und erneut an</a>.', [
```

Comment:
I guess in the current situation not preparing strings to be translatable is OK, right?

Reply:
Yes, the existing lines are just moved to another location. The inline comment (also moved) describes what the message is about.